perf/benchmarkapps/BenchmarkClient/Workers/GrpcRawUnaryWorker.cs

```diff
@@ -30,12 +30,14 @@ namespace BenchmarkClient.Workers
 {
     public class GrpcRawUnaryWorker : IWorker
     {
+        private readonly DateTime? _deadline;
         private HttpClient? _client;
 
-        public GrpcRawUnaryWorker(int id, string target)
+        public GrpcRawUnaryWorker(int id, string target, DateTime? deadline = null)
```

Comment:
A quick search of the PR diff isn't showing me where the caller to GrpcRawUnaryWorker is. Did the call site not change? Is _deadline always null here?

Reply:
It's a minor benchmark improvement I didn't want to throw away so I snuck it into this PR. It's not used yet.